client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -107,6 +129,14 @@ const DataFilesToolbar = ({ scheme }) => {
           onClick={download}
           disabled={!canDownload}
         />
+        {showMakeLink && (
+          <ToolbarButton
+            text="Link"
+            iconName="link"
+            onClick={toggleLinkModal}
+            disabled={!canDownload}
```

Comment:
do we need to additionally check here (or in line 132) to make sure that this is not a file **on a non-tapis** systems, right?